app/controllers/taxons_controller.rb

```diff
@@ -106,6 +106,12 @@ def discard_draft
     redirect_to taxons_path, success: t("controllers.taxons.discard_draft_success")
   end
 
+  def download_tagged
```

Comment:
would this be better expressed as `GET /taxons.csv` or is the functionality so different?